internal/storage/store.go

```diff
@@ -92,8 +93,8 @@ type Store interface {
 type MutableStore interface {
 	Store
 	AddOrUpdate(context.Context, ...policy.Wrapper) error
-	AddOrUpdateSchema(context.Context, string, []byte) error
-	DeleteSchema(context.Context, string) error
+	AddOrUpdateSchema(context.Context, []*schemav1.Schema) error
+	DeleteSchema(context.Context, []string) error
```

Comment:
These methods would have been nicer to use with single items (the most common case) if they accepted varargs. You don't have to change them now; just a note for the future.

```suggestion
	AddOrUpdateSchema(context.Context, ...*schemav1.Schema) error
	DeleteSchema(context.Context, ...string) error
```